substrate/pkg/controller/substrate/cluster/config.go

```diff
@@ -208,11 +207,12 @@ func (c *Config) ensureBucket(ctx context.Context, substrate *v1alpha1.Substrate
 		logging.FromContext(ctx).Infof("Created s3 bucket %s", aws.StringValue(discovery.Name(substrate)))
 	}
 	//sets tags on a bucket. Any existing tags are replaced.
-	bucket := discovery.Name(substrate)
-	svc := s3.New(session.New())
-	putInput := &s3.PutBucketTaggingInput{Bucket: bucket, Tagging: &s3.Tagging{TagSet: []*s3.Tag{{
-		Key: aws.String("kit.aws/substrate"), Value: aws.String("Kitctl")}}}}
-	svc.PutBucketTagging(putInput)
+	putInput := &s3.PutBucketTaggingInput{Bucket: discovery.Name(substrate), Tagging: &s3.Tagging{TagSet: []*s3.Tag{{
+		Key: aws.String("kit.aws/substrate"), Value: aws.String(substrate.Name)}}}}
+	_, err := c.S3.PutBucketTagging(putInput)
+	if err != nil {
+		return fmt.Errorf("adding tag %w", err)
+	}
```

Comment:
```suggestion
	if _, err := c.S3.PutBucketTagging(&s3.PutBucketTaggingInput{
	                 Bucket: discovery.Name(substrate), 
	                 Tagging: &s3.Tagging{TagSet: []*s3.Tag{{
		                              Key: aws.String("kit.aws/substrate"), 
		                              Value: aws.String(substrate.Name)},
		         }},
        }); err != nil {
		return fmt.Errorf("adding tag %w", err)
	}
```
Might need to run go fmt but gives you the general idea.